worlds/alttp/Rom.py

```diff
@@ -3,9 +3,9 @@
 import Utils
 from Patch import read_rom
 
-LTTPJPN10HASH = '03a63945398191337e896e5771f77173'
-RANDOMIZERBASEHASH = '9952c2a3ec1b421e408df0d20c8f0c7f'
-ROM_PLAYER_LIMIT = 255
+LTTPJPN10HASH: str = "03a63945398191337e896e5771f77173"
+RANDOMIZERBASEHASH: str = "9952c2a3ec1b421e408df0d20c8f0c7f"
+ROM_PLAYER_LIMIT: int = 255
```

Comment:
I don't think that adding typing here helps :thinking: Typing deduction should be possible through the assignment

Reply:
yeah, but it also doesn't hurt.